docs/en/latest/support-fips-in-apisix.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Support FIPS in APISIX
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+OpenSSL 3.0 [supports](https://www.openssl.org/blog/blog/2022/08/24/FIPS-validation-certificate-issued/)
+[FIPS](https://en.wikipedia.org/wiki/FIPS_140-2) mode. So to support
+FIPS in APISIX, we just need to compile apisix-base with OpenSSL 3.0.
+
+## Compilation
+
+```bash
+# should be as root user
+
```

Comment:
This comment could be replaced by text before the code block:

"To compile apisix-base with OpenSSL 3.0, run the commands below as root user:"